api/cmdroute/router.go

```diff
@@ -316,18 +316,42 @@ type autocompleterData struct {
 	data    discord.AutocompleteOption
 }
 
+// findAutocompleter finds the autocomplete handler for the given option name.
+// It checks the current router and its groups.
 func (r *Router) findAutocompleter(ev *discord.InteractionEvent, data discord.AutocompleteOption) (autocompleterData, bool) {
+	found, ok := r.findAutocompleterOnce(ev, data)
+	if ok {
+		return found, true
+	}
+
+	for _, sub := range r.groups {
+		found, ok = sub.findAutocompleterOnce(ev, data)
```

Comment:
Should this not be `sub.findAutocompleter()` instead?

Reply:
Does commandHandler have the same problem?

Reply:
Ah, I see what you mean! This works then, the recursive call is just done elsewhere.